### PR TITLE
TxFee: Add fees and size of the transaction in the table `transactions`

### PR DESCRIPTION
**Pull request by MichaelKim20**
I added the following fields.
Size of Transaction
Fee for Transaction
Payload Fee
I also modified the code on the endpoint where the transaction fee should be provided.
In addition to providing the fee information, they will also be used for statistical work on appropriate fees depending on the size of the transaction in the future.

Relates to #274

### CODE REVIEW

File: docs/Database_Schema.md [hunk 1]
```diff
@@ -278,7 +284,10 @@ CREATE TABLE IF NOT EXISTS information (
 | payload           | BLOB      |    | Y        |          | The transaction data payload |
 | lock_height       | INTEGER   |    | Y        |          | This transaction may only be included in a block with `block_height >= lock_height`|
 | time              | INTEGER   |    | Y        |          | Received time |
-
+| tx_fee            | INTEGER   |    | Y        |          | The fee of this transaction |
+| payload_fee       | INTEGER   |    | Y        |          | The payload fee of this transaction  |
+| tx_size           | INTEGER   |    | Y        |          | The size of this transaction  |
+|
```
Comment (TrustHenry) [inline docs/Database_Schema.md]: Unnecessary Line.

File: src/modules/storage/LedgerStorage.ts [hunk 2]
```diff
@@ -478,6 +484,53 @@ export class LedgerStorage extends Storages
...
+
+            let utxo = tx.inputs
+                .map(m => `x'${m.utxo.toBinary(Endian.Little).toString("hex")}'`);
+
+            let sql = `SELECT
+                SUM(O.amount) as sum_inputs
+            FROM
+                utxos O
+            WHERE
+                O.utxo_key in (${utxo.join(',')}); `;
```
Comment (TrustHenry) [inline src/modules/storage/LedgerStorage.ts]: It looks pretty when it's aligned.

**Review by TrustHenry — APPROVED** [review]
LGTM , few styles

**Comment by MichaelKim20** [comment]
I fixed all. Thank you for your review

**Pull request merged**